res/enkf/export/gen_data_collector.py

```diff
@@ -9,7 +9,14 @@
 
 class GenDataCollector:
     @staticmethod
-    def loadGenData(ert: EnKFMain, case_name, key, report_step, realization_index=None):
+    def loadGenData(
+        ert: EnKFMain,
+        case_name,
+        key,
+        report_step,
+        realization_index=None,
+        read_only: bool = False,
+    ):
         """@type ert: EnKFMain
         @type case_name: str
         @type key: str
```

Comment:
`@rtype` is deprecated in favor of type-hints and the docstring is missing a description of `read_only`.